cluster/gce/config-default.sh

```diff
@@ -540,11 +540,4 @@ PROXY_TAG="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
 # true if this is a resource partition cluster
 KUBERNETES_RESOURCE_PARTITION="${KUBERNETES_RESOURCE_PARTITION:-false}"
 # true if this is a tenant partition cluster
-KUBERNETES_TENANT_PARTITION="${KUBERNETES_TENANT_PARTITION:-false}"
-
-if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == true ]]; then
```

Comment:
what's the impact/reason of removing all these?

Reply:
there should be no impact after merging with Sonya's change.
@chenqianfzh thanks for cleaning this up. Please remove it from config-test.sh as well if possible (no need to retest)

Reply:
These code created two test tenants, arktos/zeta, when we were trying with 2 TPs. Yet the code was changed to create variable number of TPs, where two test tenants may not be enough. We need the tester to create the test tenants on the fly. Therefore this section of code is removed.

Reply:
thanks consider this comment resolved.